Events/Controllers/EventController.cs

```diff
@@ -29,7 +29,130 @@ public EventController()
        
 
 
-       
+        public async Task<List<Event>> RetrieveEvents()
+        {
+            List<Event> events = new List<Event>();
+         
+            var request = new ScanRequest
+            {
+                TableName = "Events"
+            };
+            var response = await _dynamoDBClient.ScanAsync(request);

```

Comment:
Another thing we can talk about later is how to design a dynamoDb table, to avoid operation like scanning the entire table.